### PR TITLE
Fix conj/get behavior

### PR DESCRIPTION
**Pull request by jmmk**
Addresses #6 and #7 

I branched this off #8, so it includes those commits as well.

I added invoke to the cljs implementation to be consistent with clj, and made sure we are using `v` and `value` to make it clear we are working with the values

### CODE REVIEW

File: build.boot [hunk 1]
```diff
@@ -1,8 +1,9 @@
 (set-env! :source-paths #{"src"}
-          :dependencies '[[adzerk/boot-test "1.0.5-SNAPSHOT" :scope "test"]
-                          [crisptrutski/boot-cljs-test "0.2.0-SNAPSHOT" :scope "test"]
+          :dependencies '[[adzerk/boot-test "1.1.1" :scope "test"]
+                          [org.clojure/clojurescript "1.7.228" :scope "test"]
+                          [crisptrutski/boot-cljs-test "0.2.1" :scope "test"]
                           [adzerk/bootlaces "0.1.13" :scope "test"]
-                          [collection-check "0.1.7-SNAPSHOT" :scope "test"]
+                          [collection-check "0.1.6" :scope "test"]
```
Comment (martinklepsch) [inline build.boot:6]: 0.1.6 does not have cljs support I think, did you successfully run tests with this setup?

Comment (jmmk) [inline build.boot:6]: The test has it only for Clojure:

``` clojure
#?(:clj
   (deftest collection-check
     (ccheck/assert-set-like
      (ccset/custom-comparator-set identity)
      clojure.test.check.generators/int)))
```

To answer your question - yes, I successfully ran the tests.

**Comment by martinklepsch** [comment]
Would be great if this PR also updates the readme section related to `get`.

**Comment by jmmk** [comment]
Forgot about the README - thanks for the reminder. Let me know if the updated description is not clear.

**Comment by martinklepsch** [comment]
@jmmk looks good to me :+1: I added you as a collaborator so feel free to go ahead and merge this.

Maybe it would make sense to also change the artifact ID to just `ccset` or `custom-comparator-set` then we can setup push access for multiple contributors.

**Pull request merged**